### ISSUE TITLE
Item has already been added - error when Run-TestsInNavContainer

### ISSUE THREAD

**Issue by jimmymcp** [microsoft/navcontainerhelper#462]
Getting the following error when calling Run-TestsInNavContainer. I can run all tests from the Test Tool page successfully.

  Codeunit 9093452 DD Tests CDDTMN Success (3.751 seconds)
  Codeunit 9093453 DD Tests - Fields CDDTMN Success (25.335 seconds)
  Codeunit 9093455 DD Tests - Config CDDTMN Success (1.21 seconds)
  Codeunit 9093456 DD Tests - Printing CDDTMN Success (0.539 seconds)
  Codeunit 0  Success (0.229 seconds)
Item has already been added. Key in dictionary: '9093456'  Key being added:
'9093456'
At C:\ProgramData\NavContainerHelper\Extensions\bc\PsTestTool\PsTestFunctions.p
s1:175 char:17
+                 $codeUnitNames += @{ $codeunitId = $codeunitName }
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (:) [], ArgumentException
    + FullyQualifiedErrorId : System.ArgumentException

**Comment by freddydk** [comment]
Do you have Codeunit 9093456 twice in the list test suite?

**Comment by jimmymcp** [comment]
Sorry for the slow response, been on hols.

No, I don't have the codeunit listed twice in the suite. If I remove that test codeunit the other 4 run fine. Consistently, if i re-add 9093456 I get the error above.

**Comment by jimmymcp** [comment]
Update: I've now realised that we had two different codeunits with that same number. They were in different branches and I've only noticed now that I've completed the pull requests - but maybe that's something to do with it?

I would have been switching between the branches and publishing them (from VS Code) to the same container to run the tests. Some metadata for both objects hanging round between publishes perhaps?
